.travis.yml

```diff
@@ -16,8 +16,6 @@ env:
   global:
     - ROS_PARALLEL_TEST_JOBS=-j1
   matrix:
-    - ROS_DISTRO=kinetic  ROS_REPO=ros              UPSTREAM_WORKSPACE=https://raw.github.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall
```

Comment:
Please move this a separate,first commit. Thanks!

Reply:
sorry for the delay, Monday was a holiday here. moved `.travis.yml` change to https://github.com/ros-controls/control_toolbox/pull/76